### ISSUE TITLE
Text doesn't appear in full in starter pack

### ISSUE THREAD

**Issue by SpriterGors** [bluesky-social/social-app#5961]
### Steps to Reproduce

1. On mobile, click on a starter pack
2. scroll to the end
3. notice that the text is cut and can't be scrolled up to be read.

### Attachments

Screenshot of my phone with the last text completely cut by the app buttons. It is impossible to scroll up.

![image](https://github.com/user-attachments/assets/e823d915-dea0-4e3f-a475-7cd8a0fcf697)

### What platform(s) does this occur on?

Android

### Device Info

Android 10, Samsung SM J6000GT

### What version of the app are you using?

Version 1.92.1.355 89cff31b6 (prod)

**Comment by khuddite** [comment]
I am looking into the issue

**Comment by khuddite** [comment]
![image](https://github.com/user-attachments/assets/4c3ac4f2-8f25-4317-af10-5c3d3e907129)

I could not recreate the issue on my phone or any Android emulators. It seems the issue is device-specific.

**Comment by SpriterGors** [comment]
Thanks for looking for me.

I looked at other starter packs with this same device but they all behaved normally. That is, until I looked at the starter pack in OP again, and lo and behold, the problem is there:

Screenshot with user "jackal chan" cut off. It is impossible to scroll down.
![Sem título](https://github.com/user-attachments/assets/b3201aeb-a86b-4536-9054-6e4bf9d653c3)

This is the starter pack link: https://go.bsky.app/FKgUvos

This is what should've appeared under "jackal chan's" bio:
![image](https://github.com/user-attachments/assets/99243911-c3ea-41d4-a884-9bc4c9265b2c)

This is the starter pack's title. Could it be that the emoji in the list title is causing this issue?
![Sem título](https://github.com/user-attachments/assets/394bc555-1d6f-4836-a12c-56cc65fcf12c)
